File: projects/inflection/modules/dev/dev_py/ru/declension/_modify.py
from projects.inflection.modules.dev.dev_py import a
from projects.inflection.modules.dev.dev_py import mw
from projects.inflection.modules.dev.dev_py import tools as _

dev_prefix = 'User:Vitalik/'  # comment this on `prod` version


from .run.parts import _prepare as _prepare
from .run.parts import _transform as _transform


module = 'modify'  # local


@a.starts(module)
def generate_parts(func, i):  # export
    _prepare.prepare(i)
    _transform.transform(i)

    _.ends(module, func)
# end


# return export

File: projects/inflection/modules/dev/dev_lua/ru/declension/_modify.lua
local dev_prefix = ''
dev_prefix = 'User:Vitalik/'  -- comment this on `prod` version

local export = {}
local _ = require('Module:' .. dev_prefix .. 'inflection/tools')


local _prepare = require('Module:' .. dev_prefix .. 'inflection/ru/declension/modify/prepare')  -- '' =  -- '_' /prepare
local _transform = require('Module:' .. dev_prefix .. 'inflection/ru/declension/modify/transform')  -- '' =  -- '_' /transform


local module = 'modify'


-- @starts
function export.modify(info)                                             --
	func = "modify"
	_.starts(module, func)

	_prepare.prepare(info)                                                    --
	_transform.transform(info)                                                --
									                                          --
	_.ends(module, func)                                                      --
end                                                                         --


return export
